engine/engine/urls.py

```diff
@@ -35,6 +35,8 @@
     path("integrations/v1/", include("apps.integrations.urls", namespace="integrations")),
     path("twilioapp/", include("apps.twilioapp.urls")),
     path("api/v1/", include("apps.public_api.urls", namespace="api-public")),
+    path("mobile_app/v1/", include("apps.mobile_app.urls", namespace="mobile_app")),
+    path("api/internal/v1/mobile_app/", include("apps.mobile_app.urls", namespace="mobile_app_tmp")),
```

Comment:
I'm wondering if `api/internal/v1/mobile_app/*` routes are actually used?

Reply:
yes, we're using 
'api/internal/v1/mobile_app/auth_token'

Reply:
and also 
'api/internal/v1/mobile_app/fcm'

Reply:
thanks. I’ll leave these untouched for now. if we want to remove a set, can do it in a future PR